owa/assembly.xml

```diff
@@ -0,0 +1,24 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3 http://maven.apache.org/xsd/assembly-1.1.3.xsd">
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <baseDirectory>dist</baseDirectory>
+    <id>zip-owa</id>
+    <formats>
+        <format>zip</format>
+    </formats>
+    <fileSets>
+        <fileSet>
+            <directory>app</directory>
+            <outputDirectory>.</outputDirectory>
+            <includes>
+                <include>build/*</include>
+                <include>css/*</include>
+                <include>img/*</include>
+                <include>admissionLocations.html</include>
+                <include>bedTags.html</include>
+                <include>bedTypes.html</include>
```

Comment:
@sanishmaharjan surely there is an ant pattern to get all .html files at the root level, rather than listing them one by one? Any reason why you wouldn't do that?

Reply:
@mks-d Done.